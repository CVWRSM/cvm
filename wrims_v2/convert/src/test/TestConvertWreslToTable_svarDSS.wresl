define evap_1 {timeseries kind 'EVAPORATION-RATE' units 'IN'}
define evap_2 {timeseries kind 'EVAPORATION-RATE' units 'IN'}
