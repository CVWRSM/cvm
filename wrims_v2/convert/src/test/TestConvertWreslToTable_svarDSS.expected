NAME,KIND,UNIT
evap_1,EVAPORATION-RATE,IN
evap_2,EVAPORATION-RATE,IN
